src/lib/viewers/controls/color-picker/__tests__/ColorPickerControl-test.tsx

```diff
@@ -1,39 +1,31 @@
 import * as React from 'react';
 import { shallow, ShallowWrapper } from 'enzyme';
-import { AnnotationMode } from '../../annotations/types';
+import { bdlBoxBlue } from 'box-ui-elements/es/styles/variables';
```

Comment:
Do we need to import this specific variable for the tests? Can it be a generic color, instead?

Reply:
We can use a generic color.